bitsv/network/services/network.py

```diff
@@ -10,6 +11,34 @@ def set_service_timeout(seconds):
     DEFAULT_TIMEOUT = seconds
 
 
+class BitIndex3Normalized(BitIndex3):
+
+    def __init__(self, api_key=None, network="main"):
+        self.api_key = api_key
+        self.network = network
+        self.headers = self._get_headers()
+        self.authorized_headers = self._get_authorized_headers()
+
+    def _get_headers(self):
+        return {
+            'Content-Type': 'application/json',
+            'Accept': 'application/json',
+        }
+
+    def _get_authorized_headers(self):
+        headers = self._get_headers()
+        headers['api_key'] = self.api_key
+        return headers
+
+    def get_transactions(self, address):
+        # override "get_transactions" namespace to return a list of txids only to keep all apis the same
```

Comment:
The `get_balance `function returns a list of txids and is far more performant than the `get_transactions` function - especially for keys that have a voluminous hx of spamming op_return data... I want the NetworkAPI bread and butter functions to be quick and responsive... once you have the txids you can always pick and choose via the `get_transaction` function.